src/controllers/bots.ts

```diff
@@ -411,9 +414,11 @@ export function buildBotPayload(msg: Msg): SphinxBot.Message {
   return m
 }
 
-export async function receiveBotRes(payload) {
+//export async function receiveBotRes(payload: Msg): Promise<void> { // TODO which type Msg? Message?
+export async function receiveBotRes(payload): Promise<void> {
   sphinxLogger.info('=> receiveBotRes', logging.Network) //, payload)
-  const dat = payload.content || payload
+  const dat = payload
+  // const dat = payload.content ||  payload // TODO
```

Comment:
just leaving a comment here for visibility, I see you have the TODO

Reply:
which type is the payload?

Reply:
The payload is created on the sending node here https://github.com/stakwork/sphinx-relay/blob/master/src/network/send.ts#L243. Would be great to get it typed

Reply:
That function returns a `Msg`, you mean payload is a `Msg`?

Reply:
yes its a `Msg`. "payload.content ||  payload" is there because of an older version of sphinx that had everything inside "content", but i think we can deprecate that now (`const dat = payload` is fine). 

Here is a fuller `Msg`:
```
export interface Msg {
  type: number
  message: {
    uuid: string
    content: string
    amount: number
    id?: number
    replyUuid?: string
    mediaToken?: string
    mediaKey?: string
    mediaType?: string
    date?: string
    originalMuid?: string
    status?: number
    remoteContent?: string
    skipPaymentProcessing?: boolean
    purchaser?: number
  }
  chat: {
    uuid: string
    type?: number
    members?: { [k: string]: any }
    name?: string
    groupKey?: string
    host?: string
  }
  sender: {
    id?: number
    pub_key: string
    alias: string
    role: number
    route_hint?: string
    photo_url?: string
  }
  network_type?: number
  isTribeOwner?: boolean
  dest?: string
  owner?: any
  bot_id?: any
  bot_name?: string
  recipient_id?: any
  action?: string
}
```

Reply:
https://github.com/stakwork/sphinx-relay/blob/msg-type/src/network/interfaces.ts

Reply:
i will merge that branch into this pull, and also resolves my new comment about missing mediaKey field